lms/envs/common.py

```diff
@@ -1732,21 +1729,9 @@
 # The name of the require.js script used by your project, relative to REQUIRE_BASE_URL.
 REQUIRE_JS = "common/js/vendor/require.js"
```

Comment:
These are all settings from django-require:

https://github.com/etianen/django-require

It is used when building the bundles with r.js.